### ISSUE TITLE
Creation of Badge for Sponsorship

### ISSUE THREAD

**Issue by adina** [swcarpentry/communications#18]
Reference https://github.com/adina/communications/issues/10

[From Greg forwarded email] I think if a little bit of HTML was provided to sponsors for use on their sites, it might drive traffic your way as well, which can only be good. Anything to lift the profile.

Something people could append to an mail signature would also be handy – it helps to get the word out.

Badges are good – they help start conversations.

**Comment by swaldman3** [comment]
I'm happy to give this a shot. 
My thought would be to encapsulate the logo with appropriate "Sponsor" or "sponsor of" text as an image hosted by us, and provide HTML to show that image as a link to SWC. Maybe two sizes of it, one for websites & one for email sigs.
Assign to me if that sounds good, and I'll try to get to it next week.

**Comment by iglpdc** [comment]
@swaldman3 Sounds good to me.

**Comment by swaldman3** [comment]
First shot of a badge is in PR #26 . 
Questions for comms leaders:

* How is the badge design itself? Any comments?
* I think that we should host the badge image on the SWC website. How do we go about putting it there somewhere? (also, if we want to put the HTML snippet there somewhere on a web page for people to copy/paste)
* thinking about the idea of doing a small version for email signatures: Do we really want to encourage people to put images in their email signatures?

Cheers
Simon.

**Comment by adina** [comment]
Merged.  Thanks Simon!   I think the HTML snippet sounds like a good way to share the badge.   I see no need for email signatures but open to comments.

**Comment by swaldman3** [comment]
I agree on the email sigs thing - was just mentioning it because Greg mentioned it in the initial email that sparked this issue.

How do I go about getting stuff onto the SWC website? (i.e. the badge) Is there a specific repo to submit a PR to, or do I need to speak to somebody, or...?

**Comment by iglpdc** [comment]
@swaldman3 Could we have some "Sponsor" ribbon or something in the badge itself, as you mentioned in previous comments. If sponsors are going to use this to sport their sponsorship, I think it'd be more appropriate. Right now it's too similar to our logo, which has other uses.

In my opinion, the badge should be clearly different from the logo, so:
- people can really distinguish themselves as sponsors, and 
- third parties don't think that sponsors represent Software Carpentry when they use the sponsor badge.

**Comment by swaldman3** [comment]
Good point. Forgot about that. In that case, does anybody know what font is used in the logo? (this is info that should go in the style guide anyway)

**Comment by swaldman3** [comment]
OK, new version is in PR #31 .
Still need to understand how to actually get this on the SWC webspace, rather than asking people to link to github.

**Comment by swaldman3** [comment]
Update, following my little outburst on the -discuss list (sorry!) and the learnings from that:

* The existing badges need fixing up. I've opened Issue #37 for that.
* I don't think that the Sponsor badge should necessarily look like them. One reason is that I don't like to throw away the work that I've done ;-), but perhaps more importantly, the sponsor badge might appear anywhere on somebody else's website and need to immediately show what it is - having the text "software carpentry" on it is thus fairly important. The other badges are a little different, in that they're clearly inspired by the general aesthetic of other Open Badges, and are more "celebratory". Hopefully that makes sense.
* I've fixed a couple of niggles with the current Sponsor badge, and shrunk it to a smaller size as suggested by a commenter. The current version is in PR #32. 
* If people agree with all of the above, I shall submit the PNG to the site repo, and ask for somebody who understands the badge-baking system (and who, unlike me at the moment, has a working python install) to set up a new badge type over there.
